src/containers/BatchEditListingPage/BatchEditListingPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect } from 'react';
+import { compose } from 'redux';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { injectIntl } from '../../util/reactIntl';
+import {
+  LISTING_PAGE_PARAM_TYPE_DRAFT,
+  LISTING_PAGE_PARAM_TYPE_NEW,
+  NO_ACCESS_PAGE_POST_LISTINGS,
+  NO_ACCESS_PAGE_USER_PENDING_APPROVAL,
+} from '../../util/urlHelpers';
+import { hasPermissionToPostListings, isUserAuthorized } from '../../util/userHelpers';
+import { NamedRedirect, Page } from '../../components';
+import TopbarContainer from '../../containers/TopbarContainer/TopbarContainer';
+
+import css from './BatchEditListingPage.module.css';
+import { initializeUppy, requestSaveBatchListings } from './BatchEditListingPage.duck';
+import BatchEditListingWizard from './BatchEditListingWizard/BatchEditListingWizard';
+
+export const BatchEditListingPageComponent = props => {
+  const { currentUser, history, intl, params, page, onInitializeUppy, onSaveBatchListing } = props;
```

Comment:
What about the 'tab' field we pass from the router?

Reply:
It's being used in src/containers/BatchEditListingPage/BatchEditListingWizard/BatchEditListingWizard.js.

BTW, I've updated the route to:
`/listing/:category/:type/:tab`